### PR TITLE
Add rtm/2.0.0

### PR DESCRIPTION
**Pull request by intelligide**
Specify library name and version:  **rtm/2.0.0**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by prince-chrismc — DISMISSED** [review]
Seems the consumer needs to configure the SIMD instructions manually...

**Review by SSE4 — APPROVED** [review]

File: recipes/rtm/all/test_package/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import os
+from conans import ConanFile, CMake
```
Comment (madebr) [inline recipes/rtm/all/test_package/conanfile.py:2]:
```suggestion
from conans import ConanFile, CMake, tools
```

File: recipes/rtm/all/test_package/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        bin_path = os.path.join("bin", "test_package")
+        self.run(bin_path, run_environment=True)
```
Comment (madebr) [inline recipes/rtm/all/test_package/conanfile.py:16]:
```suggestion
        if not tools.cross_building(self.settings):
            bin_path = os.path.join("bin", "test_package")
            self.run(bin_path, run_environment=True)
```

**Review by madebr — COMMENTED** [review]
The profile used to export or run the test_pacakge can be a cross build profile.

**Review by prince-chrismc — APPROVED** [review]

**Review by madebr — APPROVED** [review]

**Review by theirix — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**